OpenRA.Game/Settings.cs

```diff
@@ -336,9 +336,16 @@ public class ChatSettings
 		public bool ConnectAutomatically = false;
 	}
 
+	public class LobbySettings
+	{
+		public string Faction = "Random";
```

Comment:
Can you please move player color here from `PlayerSettings` too?  That is closer to the faction than the player name because the mod tilesets etc restrict the color choices differently for each mod.